.github/workflows/backend-integration-test.yml

```diff
@@ -95,7 +95,6 @@ jobs:
 
       - name: Upload integrate tests reports
         uses: actions/upload-artifact@v3
-        if: ${{ failure() && steps.integrationTest.outcome == 'failure' }}
```

Comment:
Why do we remove this line, how and where do we get the log?

Reply:
I want to upload the log regardless of whether the CI fails or not.

Reply:
This are two questions, you just only answer the first one.

Reply:
About the file location, it remains the same as before.

![image](https://github.com/datastrato/gravitino/assets/15794564/7b595ea8-9949-4378-bd77-9a5e4af4c468)
<img width="1952" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/7b7b83a2-7162-4d9e-8520-f966226c2119">

Reply:
I think it is not necessary to upload the log if the test is successful. You can add a tag to control whether do you want to upload the logs no matter it is successful or not.

Reply:
👌